bedrock/base/templates/macros-protocol.html

```diff
@@ -76,14 +76,14 @@
       {% if image_url %}
     <div class="mzp-c-split-media{% if media_class %} {{ media_class }}{% endif %}">
       {% if include_highres_image %}
-        {{ high_res_img(image_url, {'alt': image_alt, 'l10n': l10n_image, 'class': 'mzp-c-split-media-asset'}) }}
+        {{ high_res_img(image_url, {'alt': image_alt, 'l10n': l10n_image, 'class': 'mzp-c-split-media-asset', 'loading': loading}) }}
       {% else %}
         {% if image_url.startswith('https')  %}
-          <img class="mzp-c-split-media-asset" src="{{ image_url }}" alt="{{ image_alt }}">
+          <img class="mzp-c-split-media-asset" src="{{ image_url }}" alt="{{ image_alt }}" loading="{{ loading }}">
         {% elif l10n_image %}
-          <img class="mzp-c-split-media-asset" src="{{ l10n_img(image_url) }}" alt="{{ image_alt }}">
+          <img class="mzp-c-split-media-asset" src="{{ l10n_img(image_url) }}" alt="{{ image_alt }}", loading="{{ loading }}">
```

Comment:
Please remove the commas in the `<img>` elements, as they are not needed to separate attributes and are creating invalid markup issues.

Reply:
Oh wow, my mistake! Can't believe I missed that.